clients/java/src/main/java/io/camunda/client/api/command/UpdateJobCommandStep1.java

```diff
@@ -16,7 +16,7 @@
 package io.camunda.client.api.command;
 
 import io.camunda.client.api.response.UpdateJobResponse;
-import io.camunda.client.protocol.rest.JobChangeset;
+import io.camunda.client.api.search.JobChangeset;
```

Comment:
🔧 This changeset is only used in a command, not in search. We should put those client DTOs into the packages they are related to, if possible.